src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -12,6 +12,12 @@ namespace Raven.Server.Documents
 {
     public class TombstoneCleaner : BackgroundWorkBase
     {
+        private readonly ITombstoneAware.TombstoneType[] _tombstoneTypes = new ITombstoneAware.TombstoneType[]
+        {
+            ITombstoneAware.TombstoneType.Documents,
+            ITombstoneAware.TombstoneType.TimeSeries
```

Comment:
What about counters?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-14816 we do not clean them at all